src/main/java/uk/co/ramp/covid/simulation/lockdown/LockdownController.java

```diff
@@ -1,36 +1,30 @@
 package uk.co.ramp.covid.simulation.lockdown;
 
 import uk.co.ramp.covid.simulation.util.Time;
-import uk.co.ramp.covid.simulation.population.Population;
-
 import java.util.ArrayList;
 import java.util.List;
 
 public class LockdownController {
-    private final Population population;
-    private List<LockdownEvent> events;
-    private List<LockdownEventGenerator> eventGenerators;
+    private final List<LockdownEvent> events;
+    private final List<LockdownEventGenerator> eventGenerators;
 
-    public LockdownController(Population p) {
-        population = p;
+    public LockdownController() {
         events = new ArrayList<>();
         eventGenerators = new ArrayList<>();
     }
     
-    public LockdownController addComponent(LockdownEvent c) {
```

Comment:
The idea here was presumably so that tests could do something like:
```java
pop.getLockdownController()
    .addComponent(new FullLockdown(...)
    .addComponent(new XYZ...)
    .addComponent(new ABC...);
```
rather than
```java
pop.getLockdownController().addComponent(new FullLockdown(...)
pop.getLockdownController().addComponent(new XYZ...)
pop.getLockdownController().addComponent(new ABC...);
```
but I guess that the fact that it hasn't been used suggests that it is not needed, and there's also the option to do:
```java
var lc = pop.getLockdownController();
lc.addComponent(new FullLockdown(...);
lc.addComponent(new XYZ...);
lc.addComponent(new ABC...);
```